packages/sources/coinbase-prime/src/endpoint/balance.ts

```diff
@@ -0,0 +1,45 @@
+import { AdapterEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { InputParameters } from '@chainlink/external-adapter-framework/validation'
+import { SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
+import { config } from '../config'
+import { httpTransport } from '../transport/balance'
+
+export const inputParameters = new InputParameters(
```

Comment:
I guess post-v2 adapters we don't have any standard input params for balance endpoints?